components/menu-dropdown/__examples__/custom-trigger.jsx

```diff
@@ -20,6 +20,10 @@ class Example extends React.Component {
 						{ label: 'Menu Item Three', value: 'C0' },
 						{ type: 'divider' },
 						{ label: 'Menu Item Four', value: 'D0' },
+						{ label: 'Menu Item Five', value: 'E0' },
+						{ label: 'Menu Item Six', value: 'F0' },
+						{ type: 'divider' },
+						{ label: 'Menu Item Seven', value: 'G0' },
```

Comment:
Added more items so that the dropdown scrollbar will be visible in storybook examples